ui/app/src/utils/content.ts

```diff
@@ -1090,14 +1089,15 @@ export function applyAssetNameRules(name: string, options?: { allowBraces: boole
  * letter, number, dash or underscore.
  */
 export function applyContentNameRules(name: string): string {
-  return slugify(name, {
-    lower: true,
-    // Setting `strict: true` would disallow `_`, which we don't want.
-    strict: false,
-    // Because of the moment where the library trims, `trim: true` caused undesired replacement of `-`
-    // at the beginning or end of the slug.
-    trim: false
-  }).replace(/[^a-z0-9-_]/g, '');
+  return (
+    // Replace accented vowels with their non-accented counterpart
+    replaceAccentedVowels(name)
+      // replace spaces with dashes
+      .replace(/\s+/g, '-')
+      .toLowerCase()
+      // remove any character that is not a letter, number, dash or underscore, and allow braces if specified
+      .replace(/[^a-zA-Z0-9-_]/g, '')
+  );
```

Comment:
Can't possibly replace all of slugify with just this. You need to do due diligence and find out what's preventing 2 hyphens inside the slugifyer.